src/r5py/r5/travel_time_matrix_computer.py

```diff
@@ -92,10 +100,24 @@ def __init__(
             transport_network = TransportNetwork(*transport_network)
         self.transport_network = transport_network
 
+        # Quick validation to ensure an ID column exists
+        if "id" not in origins.columns:
+            raise NoIDColumnError("Origin dataset must contain an 'id' column.")
+        # And to make sure the column is indeed unique
+        if len(origins.id.unique()) < origins.shape[0]:
+            raise NotUniqueIDError("Origin id values must be unique.")
         self.origins = origins
 
         if destinations is None:
             destinations = origins
+
+        # Quick validation to ensure an ID column exists
```

Comment:
Could you take this entire block into a utility function that can be called from `origins()` and from `destinations()`, and possibly be reused in other classes in the future?

Reply:
This utility function could be a private method of whatever should be the parent class of all ‘worker classes’, but for the time being a separate file in `utils/` would probably do the job.

Reply:
@christophfink @wklumpen: Let's turn this into a separate issue, so that we can get this PR rolling.